src/main/scala/com/advancedtelematic/campaigner/db/Campaigns.scala

```diff
@@ -169,8 +190,8 @@ protected [db] class Campaigns(implicit db: Database, ec: ExecutionContext)
     io.transactionally
   }
 
-  def create(campaign: Campaign, groups: NonEmptyList[GroupId], metadata: Seq[CampaignMetadata]): Future[CampaignId] =
-    campaignRepo.persist(campaign, groups.toList.toSet, metadata)
+  def create(campaign: Campaign, groups: NonEmptyList[GroupId], devices: Set[DeviceId], metadata: Seq[CampaignMetadata]): Future[CampaignId] =
```

Comment:
We could get rid of the groups here right? Since we're passing all the devices directly.

Reply:
Not exactly. We return groups in `GetCampaign` and FE depends on this information. Luckily when they adapt to new stats, we can remove the field from `GetCampaign` and forget about groups completely.